pppd/options.c

```diff
@@ -1066,8 +1054,32 @@ print_options(printer_func printer, void *arg)
 static void
 usage(void)
 {
-    if (phase == PHASE_INITIALIZE)
-	fprintf(stderr, usage_string, VERSION, progname);
+    FILE *fp = stderr;
+    if (phase == PHASE_INITIALIZE) {
+        fprintf(fp, "%s v%s\n", PACKAGE_NAME, PACKAGE_VERSION);
+        fprintf(fp, "Copyright (C) 1999-2022 Paul Mackerras. All rights reserved.\n\n");
```

Comment:
Perhaps say "Paul Mackerras and others" rather than just "Paul Mackerras".

Reply:
Fine by me.